pkgs/applications/version-management/pijul/default.nix

```diff
@@ -4,17 +4,13 @@ with rustPlatform;
 
 buildRustPackage rec {
   name = "pijul-${version}";
-  version = "0.10.0";
+  version = "0.11.0";
 
   src = fetchurl {
     url = "https://pijul.org/releases/${name}.tar.gz";
-    sha256 = "1lkipcp83rfsj9yqddvb46dmqdf2ch9njwvjv8f3g91rmfjcngys";
+    sha256 = "e60793ab124e9054c1d5509698acbae507ebb2fab5364d964067bc9ae8b6b5e5";
```

Comment:
Note that this is the sha256 published at https://pijul.org/downloads/
Should I convert this to base32?

Reply:
@RohanHart 
If you want.

You can use ` nix-hash --to-base32 --type sha256`.